### ISSUE TITLE
Consider adding the resize handles to the Post Editor

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#47361]
## What problem does this address?

In chatting with @elmastudio and a few others, the topic came up around adding the resizable view currently available in template part focus mode to the post editor:

Post editor view:

https://user-images.githubusercontent.com/26996883/214141157-a56cd09b-3bea-467c-865e-5746bb6c3790.mov

Focus mode view:

https://user-images.githubusercontent.com/26996883/214140847-27ee7f91-597e-4b61-8a6d-342a82e027e5.mov

## What is your proposed solution?

Bring the focus mode resizable option to the post editor to have a more unified and modern approach. This is not a good term but perhaps there's just one menu item under Preview that brings you to the draggable handle view or perhaps simply clicking "preview" takes you there. Here are a few ideas for fun:

<img width="402" alt="Screen Shot 2023-01-23 at 12 19 12 PM" src="https://user-images.githubusercontent.com/26996883/214141442-fbf0d4e9-6bb0-4911-beb3-0eeb2ab352f2.png">

<img width="391" alt="Screen Shot 2023-01-23 at 12 20 27 PM" src="https://user-images.githubusercontent.com/26996883/214141647-a02a0b56-3593-4202-a8d0-41e42f69bc73.png">

<img width="387" alt="Screen Shot 2023-01-23 at 12 21 12 PM" src="https://user-images.githubusercontent.com/26996883/214141783-97195763-02e6-4cbc-9ed6-cc556a6ce330.png">

Related to this, would be neat to add snapping too: https://github.com/WordPress/gutenberg/issues/47360

**Comment by ktmn** [comment]
Yes! I want all editors to have the resize handles, so much more convenient. 

Any code snippets to accomplish this?
